src/features/auth/components/AuthenticationButton/AuthenticationButton.native.test.tsx

```diff
@@ -21,4 +21,20 @@ describe('<AuthenticationButton />', () => {
 
     expect(navigate).toBeCalledWith('SignupForm', undefined)
   })
+  it('should prevent cancellation when asked for login', async () => {
```

Comment:
Tu peux ajouter un saut de ligne entre chaque it ? 
Ça aère les tests, c'est plus facile a lire 😊